ultravox/training/train.py

```diff
@@ -131,6 +132,7 @@ def main() -> None:
             name=args.exp_name,
             dir="runs",
             tags=args.run_tags,
+            save_code=True,
```

Comment:
Also enabled the `save_code=True` option for W&B that lets a `diff.patch` of the current run to be stored in W&B for better future reproducibility (tbh this should only be useful for interactive runs where the current Git working tree is dirty).